mappings/net/minecraft/structure/rule/PosRuleTestType.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/class_4996 net/minecraft/structure/rule/PosRuleTestType
+	METHOD method_26409 register (Ljava/lang/String;Lnet/minecraft/class_4996;)Lnet/minecraft/class_4996;
+		ARG 0 id
+		ARG 1 object
```

Comment:
same as your other pr, `o` or `object`?

Reply:
This is the registered stuff, i'd call it object